### DISCUSSION TITLE
有没有做点硬件生意的

### DISCUSSION THREAD

**Discussion by xiaoheicae** [filecoin-project/community-china#159] [矿机配置]
生意不好做，有没有要进点硬件的，成本价，欢迎联系13049808966

**Comment by Chenhongwe** [comment]
Fil算力填充，240/T，包半年运维，送七牛商业存储软件一年，坐标杭州，欢迎骚扰 phone：15255579473（同V）

**Comment by zhaobo443** [comment]
小型机房建设，专业IPFS节点搭建，分币系统app定制全套filecoin,包括测ceph存储技术，长期合作全部免费奉送，长期维护只抽取5%服务费！欢迎电话咨询：19965407481
